### PR TITLE
32 more bytes

### PR DESCRIPTION
**Pull request by mejango**
Add 32 more bytes to the metadata space for generic use by other delegates _before_ the run-on metadata used for this delegate.

### CODE REVIEW

File: contracts/JBTiered721Delegate.sol [hunk 1]
```diff
@@ -546,10 +546,11 @@ contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Ownable {
     bool _dontOverspend;
 
     // Skip the first 32 bytes which are used by the JB protocol to pass the paying project's ID when paying from a JBSplit.
+    // Skip another 32 bytes reserved for generic extension parameters.
     // Check the 4 bytes interfaceId to verify the metadata is intended for this contract.
     if (
       _data.metadata.length > 36 &&
```
[the remark this answers is not in the document]
Comment (mejango) [reply contracts/JBTiered721Delegate.sol:552]: 68?

[the remark this answers is not in the document]
Comment (drgorillamd) [reply contracts/JBTiered721Delegate.sol:551]: Skipping 2x32 + 4 instead

[the remark this answers is not in the document]
Comment (mejango) [reply contracts/JBTiered721Delegate.sol:551]: oo. _data.metadata.length > 68, you mean. gotcha, same page.

**Review by drgorillamd — APPROVED** [review]

**Comment by mejango** [comment]
lets get redemption scoot in here too @xBA5ED @drgorillamd

**Pull request merged**